spec/models/game_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  it "should populate the board with pieces when started" do
+    game = FactoryGirl.create(:game)
+    expect(game.pieces.count).to eq 32
+  end
```

Comment:
Let's wait for Gary to start the "Add a Piece Model" task before adding this test.